client/src/Components/Recipe/DisplayReviews.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react'
+import { Review } from '../../types/types';
+import axios from 'axios';
+import PostReview from '../PostReview/PostReview';
+import ReviewsList from '../Util/ReviewsList/ReviewsList';
+
+// --------------------------------------------------
+
+const serverBaseUrl = "http://localhost:4000";
+
+// ==================================================
+
+
+function DisplayReviews(props: any) {    
+    const [reviewsData, setReviewsData] = useState([] as Review[]);
+    const [loading, setLoading] = useState(true);
+  
+    useEffect(() => {
+      fetchReviews();
+    }, []);
+
+    useEffect(() => {
+        setLoading(false);
+    }, [reviewsData])
+  
+    // ----------
+  
+    /**
+     * Retrieves the reviews under an associated recipeId
+     */
+    function fetchReviews() {
+      console.log(`Fetching review under recipe ${props.reviewId}.`);
+  
+      axios
+        .get(`${serverBaseUrl}/reviews?recipeId=${props.reviewId}`)
+        .then((response) => {
+          setReviewsData(response.data?.ReviewPost);
```

Comment:
Note that there might be a bug when the response does not have the `ReviewPost` key.

Reply:
Did you test this?
The key should be `ReviewPosts`, which is an object that contains the key `items` and maybe `LastEvaluatedKey`.

Reply:
Supposedly this works.